contracts/FlowCallbackScheduler.cdc

```diff
@@ -71,7 +75,7 @@ access(all) contract FlowCallbackScheduler {
     /// Scheduled callback contains methods to cancel the callback and obtain the status. 
     /// It can only be created by the scheduler contract to prevent spoofing.
     access(all) struct ScheduledCallback {
-        access(self) let scheduler: Capability<auth(CancelCallback, ReadCallbackStatus) &SharedScheduler>
+        access(self) let scheduler: Capability<auth(CancelCallback) &SharedScheduler>
```

Comment:
There was no need for the `ReadCallbackStatus` entitlement since it was just a view function